src/calculateRentalCost.js

```diff
@@ -28,6 +28,17 @@
  */
 
 function calculateRentalCost(days) {
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+  const perDayRent = 40;
+
+  if (days >= 1 && days < 3) {
+    return days * perDayRent;
+  } else if (days >= 3 && days < 7) {
+    return days * perDayRent - threeDaysDiscount;
+  } else if (days >= 7) {
+    return days * perDayRent - sevenDaysDiscount;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/206669201-dbd867f7-ebbd-449b-8a7a-d43e160558c0.png)
Let's change your code according to these rules so that your code looks cleaner